ontology-assets/ontology_info.json

```diff
@@ -1,4 +1,64 @@
 {
+  "5.2.0+experimental": {
+    "ontologies": {
+      "CL": {
+        "version": "v2024-08-16",
+        "source": "https://github.com/obophenotype/cell-ontology/releases/download",
+        "filename": "cl.owl"
+      },
+      "EFO": {
+        "version": "v3.69.0",
+        "source": "https://github.com/EBISPOT/efo/releases/download",
+        "filename": "efo.owl"
+      },
+      "HANCESTRO": {
+        "version": "3.0",
+        "source": "https://github.com/EBISPOT/hancestro/raw",
+        "filename": "hancestro.owl"
+      },
+      "HsapDv": {
+        "version": "v2024-05-28",
+        "source": "https://github.com/obophenotype/developmental-stage-ontologies/releases/download",
+        "filename": "hsapdv.owl"
+      },
+      "MONDO": {
+        "version": "v2024-08-06",
+        "source": "https://github.com/monarch-initiative/mondo/releases/download",
+        "filename": "mondo.owl"
+      },
+      "MmusDv": {
+        "version": "v2024-05-28",
+        "source": "https://github.com/obophenotype/developmental-stage-ontologies/releases/download",
+        "filename": "mmusdv.owl"
+      },
+      "NCBITaxon": {
+        "version": "v2023-06-20",
+        "source": "https://github.com/obophenotype/ncbitaxon/releases/download",
+        "filename": "ncbitaxon.owl.gz"
+      },
+      "UBERON": {
+        "version": "v2024-08-07",
+        "source": "https://github.com/obophenotype/uberon/releases/download",
+        "filename": "uberon.owl"
+      },
+      "PATO": {
+        "version": "v2023-05-18",
+        "source": "https://github.com/pato-ontology/pato/raw",
+        "filename": "pato.owl"
+      },
+      "FBbt": {
+        "version": "v2024-10-17",
+        "source": "https://github.com/FlyBase/drosophila-anatomy-developmental-ontology/releases/download",
+        "filename": "fbbt.owl"
+      },
+      "FBdv": {
+        "version": "v2024-10-17",
+        "source": "https://github.com/FlyBase/drosophila-developmental-ontology/releases/download",
+        "filename": "fbdv.owl"
+      }
+    },
+    "deprecated_on": "2024-10-21"
```

Comment:
the ontology builder is [running](https://github.com/chanzuckerberg/cellxgene-ontology-guide/actions/runs/11449981131) but for some reason when it checks for the latest schema version [here](https://github.com/chanzuckerberg/cellxgene-ontology-guide/blob/main/tools/ontology-builder/src/all_ontology_generator.py#L347) it picks up 5.2.0 and marks 5.2.0+experimental for deprecation, which isn't right. Not sure why at a glance, the check for most up-to-date semver looks right--can you look into it?

Reply:
@joyceyan I think this might actually be how the sort is supported to rank them. What do you think of`5.2.1+experimental`? also gonna start on the zebrafish equivalent pr

Reply:
i tried removing this first, and it still wasn't picking up `5.2.0+experimental` as the latest. but i think just calling it `5.2.1+experimental` might work / is potentially easier

Reply:
Maybe I should use a hyphen. 

Semver says:

> A pre-release version MAY be denoted by appending a hyphen and a series of dot separated identifiers immediately following the patch version. Identifiers MUST comprise only ASCII alphanumerics and hyphen [0-9A-Za-z-]. Pre-release versions satisfy but have a lower precedence than the associated normal version. Examples: 1.0.0-alpha, 1.0.0-alpha.1, 1.0.0-0.3.7, 1.0.0-x.7.z.92.